backend/x86_binary_emitter.ml

```diff
@@ -1836,9 +1848,9 @@ let imm arg = match arg with Imm n -> Int64.to_int n | _ -> assert false
 
 let assemble_instr b loc = function
   | ADD (src, dst) -> emit_ADD b dst src
-  | ADDSD (src, dst) -> emit_addsd b dst src
+  | ADDSD (src, dst) -> emit_add_float ~width:Cmm.Float64 b dst src
```

Comment:
Not for this PR, but I wonder
whether the size should actually
be part of the instruction (_i.e._
have only one contructor for
`addsd` and `addss`).